src-built-in/components/toolbar/src/dynamicToolbar.jsx

```diff
@@ -20,28 +20,29 @@ import WorkspaceMenuOpener from "../components/WorkspaceMenuOpener"
 import Search from "../components/Search"
 import DragHandle from "../components/DragHandle"
 
-// Support Dynamically Loading External Components
-var customComponents = [];
-customComponents["AutoArrange"] = AutoArrange;
-customComponents["AlwaysOnTop"] = AlwaysOnTop;
-customComponents["BringToFront"] = BringToFront;
-customComponents["MinimizeAll"] = MinimizeAll;
-customComponents["WorkspaceMenuOpener"] = WorkspaceMenuOpener;
-customComponents["Search"] = Search;
 
 // Styles
 import "../toolbar.css";
 import "../../../../assets/css/font-finance.css";
 import "../../../../assets/css/finsemble.css";
 
-var pinnableItems = {
-	"componentLauncher": FinsembleButton,
-	"workspaceSwitcher": WorkspaceLauncherButton
-};
-
 export default class Toolbar extends React.Component {
 	constructor(props) {
 		super(props);
+		// Able to load external components by overriding the class
+		this.customComponents = [];
```

Comment:
Why use an array here when we're using strings as indexes? This will greatly confuse folks.